pipelines/rj_cor__precipitacao_alertario/tasks.py

```
@@ -48,7 +48,7 @@ def download_alertario_data_task() -> Tuple[pd.DataFrame, pd.DataFrame]:
     http_ok = 200
 
     try:
-        response = requests.get(url, timeout=30)
+        response = requests.get(url, timeout=30, verify=False)
```

Ruff (0.15.17)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)